cherry_picker/cherry_picker.py

```diff
@@ -277,7 +277,7 @@ def amend_commit_message(self, cherry_pick_branch):
                 click.echo(cpe.output)
         return updated_commit_message
 
-    def push_to_remote(self, base_branch, head_branch, commit_message=""):
+    def push_to_remote(self, base_branch, head_branch, commit_message):
```

Comment:
Why is removing the default necessary?

Reply:
@webknjaz can we keep the original behavior and let the default commit message?

Reply:
I don't remember, it was over a year ago. But I think I wanted to prevent situations when a caller would accidentally leave it empty. Why would one want this behavior to be implicit? If a caller wants an empty title, they should state that explicitly instead of relying on the magical empty value. Besides, it's passed to `create_gh_pr()` without changes, which already enforces the same.